src/coreclr/vm/ceeload.cpp

```diff
@@ -414,6 +414,8 @@ void Module::Initialize(AllocMemTracker *pamTracker, LPCWSTR szName)
 
     m_loaderAllocator = GetAssembly()->GetLoaderAllocator();
     m_pSimpleName = m_pPEAssembly->GetSimpleName();
+    m_path = m_pPEAssembly->GetPath();
```

Comment:
`PEAssembly::GetPath` returns the empty string if the image is in a bundle. is that what we want to cache here?

Reply:
>  Is there something else you expect it to be?

I think I'm mostly curious what the relationship between PEAssembly::GetPath() and PEImage::m_path is.  It looks like for bundled assemblies PEImage::OpenImage gets passes a non-empty path, but PEAssembly::GetPath intentionally doesn't want to return it.